edx-platform/pearson-pathways-theme/lms/templates/logout.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+{% load staticfiles theme_pipeline %}
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Logout</title>
+
+    {% javascript 'main_vendor' %}
```

Comment:
I see that this is handled inside a block in 'main_django.html'. Why didn't you considered it here?
```suggestion
    {% block main_vendor_js %}
    {% javascript 'main_vendor' %}
    {% endblock %}
```

Reply:
main_django.html brings the footer and navbar from main_django.html. In this case, I don't want that. This is not an nested template so by default I don't need to block content for its use by another template.